paternoster/paternoster.py

```diff
@@ -168,3 +207,62 @@ def execute(self):
         if status and self._success_msg:
             print(self._success_msg)
         return status
+
+    @staticmethod
+    def prompt(text):
+        """
+        Return user input from a prompt with *text*.
+
+        Works with Python 2 and 3.
+
+        """
+        try:
+            return raw_input(text)
+        except NameError:
+            return input(text)
+
+    @staticmethod
+    def get_input(param):
+        """
+        Return user input for *param*.
+
+        The text for the prompt is taken from `param['prompt']`. You can set
+        additional arguments in `param['prompt_options']`:
+
+        :strip: if `True`: strips user input
+        :empty: if `True`: allows empty input
+        :confirm: if `True` or string: prompt user for confirmation
+        :confirm_error: if string: used on confirmation error
+
+        Raises:
+            ValueError: if input and confirmation do not match.
+
+        """
+        prompt = param.get('prompt')
+        if not isinstance(prompt, six.string_types):
+            prompt = 'Input: '
```

Comment:
Maybe default this to `ParameterName: ` (name=username ➡️ `Username: `) to give a nicer experience by default?